pkg/sdk/views_gen.go

```diff
@@ -189,34 +213,31 @@ type DescribeViewOptions struct {
 	view     bool                   `ddl:"static" sql:"VIEW"`
 	name     SchemaObjectIdentifier `ddl:"identifier"`
 }
-
-// TODO [SNOW-965322]: extract common type for describe
```

Comment:
what happened with this TODO?

Reply:
It was removed by sdk generator. Moved to def file.